help_to_heat/portal/download_views.py

```diff
@@ -77,12 +77,20 @@ def add_extra_row_data(referral):
     eligibility = calculate_eligibility(row)
     epc_date = row.get("epc_date")
     epc_rating = row.get("epc_rating")
+    accept_suggested_epc = row.get("accept_suggested_epc")
     created_at = referral.created_at.astimezone(london_tz)
+    contact_number = row.get("contact_number")
+    contact_number = '="' + contact_number + '"'
+    uprn = row.get("uprn")
+    uprn = '="' + str(uprn) + '"' if uprn else ""
     row = {
         **row,
+        "contact_number": contact_number,
+        "uprn": uprn,
         "ECO4": "Energy Company Obligation 4" in eligibility and "Yes" or "No",
         "GBIS": "Great British Insulation Scheme" in eligibility and "Yes" or "No",
         "epc_rating": epc_rating and epc_rating != "Not found" or "",
+        "accept_suggested_epc": accept_suggested_epc and accept_suggested_epc != "Not found" or "",
```

Comment:
We should check with the client what the suppliers want here rather than assuming - "Not found" might be useful information (PC-400 was just because the date field needed to be empty when missing so that Excel autoformatted it as a date).

Reply:
Just confirmed with Ladun that we should keep "Not found" if the rest of the column won't contain numbers, so I'll remove this change!